src/wast-parser.cc

```diff
@@ -3527,8 +3527,10 @@ Result WastParser::ParseModuleCommand(Script* script, CommandPtr* out_command) {
       options.features = options_->features;
       Errors errors;
       const char* filename = "<text>";
-      ReadBinaryIr(filename, bsm->data.data(), bsm->data.size(), options,
-                   &errors, module);
+      if (options_->validate_binary_modules) {
+        ReadBinaryIr(filename, bsm->data.data(), bsm->data.size(), options,
+                     &errors, module);
```

Comment:
Based on the code change, this is more like ignoring the module. Not validating usually means the module is processed, but the errors are ignored.
(Note: I am not a reviewer)

Reply:
The script_module and its binary data are still made available (see the unit test for example usage), it's just the structured wabt::Module that is not provided.

Reply:
In that case perhaps a better name for this option might be "parse_binary_modules" or "no_parse_binary_modules"?  

BTW, how do invalid binaries modules in the test suite currently work without this option?

Reply:
Renamed to "parse_binary_modules".

Currently invalid modules are parsed and fail validation in WABT, which causes the test to pass. This change does not alter that behavior.

The use case is when you're using WABT to parse WAST files, but you don't want WABT to reject invalid binary modules. Maybe you want to send that invalid module data to a WASM engine that you're building. With this change, you can set `parse_binary_modules` to false and extract that invalid data out without WABT rejecting it.